include/openthread/border_routing.h

```diff
@@ -118,6 +118,21 @@ typedef struct otBorderRoutingPrefixTableEntry
     uint32_t                   mPreferredLifetime;   ///< Preferred lifetime of the on-link prefix when `mIsOnLink`.
 } otBorderRoutingPrefixTableEntry;
 
+/**
+ * Represents a group of info when platform generated RA message processed.
+ *
+ * The entries in the discovered table track the Prefix/Route Info Options in the received Router Advertisement messages
+ * from other routers on the infrastructure link.
+ *
+ */
+typedef struct otPdProcessedRaInfo
+{
+    bool    mRequirePD;               ///< Indicates whether the platform require PD
+    int64_t mNumPlatformRaReceived;   ///< The number of platform generated RA handled by ApplyPlatfromGeneratedRa
+    int64_t mNumPlatformPIOProcessed; ///< The number of PIO processed for adding OMR prefixes
```

Comment:
Do we need the counter to be `int`? Can we define it as `uint`? If we use `int64` we should mention what does negative values mean?

Reply:
Can we use `uint32_t` for these? 
(Even if we receive one every 100 msec, it will take > 10 years to reach `uint32_t` max.

Reply:
Done, thanks